packages/flutter_tools/lib/src/commands/attach.dart

```diff
@@ -21,6 +24,7 @@ import '../run_hot.dart';
 import '../runner/flutter_command.dart';
 
 final String ipv4Loopback = InternetAddress.loopbackIPv4.address;
+const String _kDartObservatoryName = '_dartobservatory._tcp.local';
```

Comment:
The naming here is inconsistent.  The `ipv4Loopback` and `ipv6Loopback` variables should start with 'k', according to the style guide, since these are globals. Of course, that may be a breaking change, but it's odd to have the observatory name using one style, and those using another.

Since you're not using it anywhere else, I'd move this constant inside of `mdnsQueryDartObservatoryPort` and rename it to `dartObservatoryName`.

Reply:
Done

Reply:
Did you not push the change yet? I don't see that it moved...